hello_nexus/without_service_definition/app.py

```diff
@@ -0,0 +1,117 @@
+"""
+This file demonstrates running a nexus service in the same namespace as the workflow that
+is calling the nexus service, without specifying a separate service definition.
+"""
+
+from __future__ import annotations
+
+import asyncio
+import uuid
+from typing import Optional
+
+from nexusrpc.handler import (
+    OperationHandler,
+    StartOperationContext,
+    operation_handler,
+    service_handler,
+)
+from temporalio import workflow
+from temporalio.client import Client
+from temporalio.nexus.handler import (
+    WorkflowOperationToken,
+    WorkflowRunOperationHandler,
+    start_workflow,
+)
+from temporalio.worker import UnsandboxedWorkflowRunner, Worker
+from temporalio.workflow import NexusClient
+
+NAMESPACE = "my-namespace"
+TASK_QUEUE = "my-task-queue"
+NEXUS_ENDPOINT = "my-nexus-endpoint"
+
+#
+# Handler
+#
+
+
+@workflow.defn
+class HandlerWorkflow:
+    @workflow.run
+    async def run(self, message: str) -> str:
+        return f"Hello {message} from workflow run operation!"
+
+
+# Here we define a nexus service by providing a service handler implementation without a
+# service contract. This nexus service has one operation.
+@service_handler
```

Comment:
I think if we're willing to have multiple service impl approaches in the basic sample for operation handlers, we can toss this in there too. I think the basic sample having 3 different services implemented different ways is more reasonable than 1 sample with 2 different service approaches and another with a third.

Alternatively, if we keep this separate, I think we should consider breaking files out similarly to the basic sample.